pkg/resources/aws/aws.go

```diff
@@ -806,7 +806,7 @@ func ListSubnets(cloud fi.Cloud, vpcID, clusterName string) ([]*resources.Resour
 
 		rtRequest := &ec2.DescribeRouteTablesInput{}
 		rtResponse, err := c.EC2().DescribeRouteTables(rtRequest)
-		if err != nil {
+		if err != nil && awsup.AWSErrorCode(err) != "InvalidRouteTableID.NotFound" {
```

Comment:
the first `for` loop below this will panic because `rtResponse` will be nil. same with the other change.

Reply:
good catch, fixed